packages/devtools_app/lib/src/screens/memory/panes/diff/widgets/snapshot_list.dart

```diff
@@ -132,6 +100,43 @@ class _SnapshotListTitle extends StatelessWidget {
   }
 }
 
+class _SnapshotListItems extends StatelessWidget {
```

Comment:
this widget should be stateful and this is where we should be creating the scroll controller and listening to the snapshot list.